libs/lume/scripts/build/build-release-notarized.sh

```
@@ -64,23 +64,77 @@ log "normal" "Ensuring .release directory exists and is accessible"
+  exit 1
+fi
+
+# --- Sign the .app bundle ---
+log "essential" "Signing .app bundle..."
+log "essential" "Using signing identity: $CERT_APPLICATION_NAME"
+
+# Ensure build.keychain is in the search list for codesign
+KEYCHAIN_PATH="$HOME/Library/Keychains/build.keychain-db"
+if [ -f "$KEYCHAIN_PATH" ]; then
+  log "essential" "Adding build keychain to search list..."
+  security list-keychains -d user -s "$KEYCHAIN_PATH" $(security list-keychains -d user | tr -d '"')
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)